validator/package/validator.yaml

```diff
@@ -10,12 +10,12 @@ wallet-dir: /var/lib/prysm/validator
 # wallet-password-file: /var/lib/prysm/validator/password.txt
 
 # Optional tuning parameters
-# For full list, see https://docs.prylabs.network/docs/prysm-usage/parameters
+# For full list, see https://agora-cl-docs.bosagora.org/docs/prysm-usage/parameters
```

Comment:
Sorry, Does not work the link.

Reply:
Good catch, I forgot that some folders and pages were renamed.

Reply:
I have dropped the commit with the other renames from this PR as it will be tackled by issue https://github.com/bosagora/AgoraMainNet/issues/80 where some pages will be removed and updated.